无效脚本命令check-commit

### Reproduction link

[https://github.com/ant-design/ant-design/blob/master/package.json](https://github.com/ant-design/ant-design/blob/master/package.json)

### Steps to reproduce

pnpm check-commit

### What is expected?

运行pnpm check-commit项目目录能找到scripts/check-commit.ts脚本。现在缺失

### What is actually happening?

➜  ant-design git:(master) pnpm check-commit

> antd@6.1.0 check-commit /Users/lemonade/Downloads/github/ant-design
> tsx scripts/check-commit.ts


node:internal/modules/run_main:122
    triggerUncaughtException(
    ^
Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/Users/lemonade/Downloads/github/ant-design/scripts/check-commit.ts' imported from /Users/lemonade/Downloads/github/ant-design/
    at finalizeResolution (node:internal/modules/esm/resolve:275:11)
    at moduleResolve (node:internal/modules/esm/resolve:860:10)
    at defaultResolve (node:internal/modules/esm/resolve:984:11)
    at nextResolve (node:internal/modules/esm/hooks:748:28)
    at resolveBase (file:///Users/lemonade/Downloads/github/ant-design/node_modules/.pnpm/tsx@4.21.0/node_modules/tsx/dist/esm/index.mjs?1765707025996:2:3744)
    at resolveDirectory (file:///Users/lemonade/Downloads/github/ant-design/node_modules/.pnpm/tsx@4.21.0/node_modules/tsx/dist/esm/index.mjs?1765707025996:2:4243)
    at resolveTsPaths (file:///Users/lemonade/Downloads/github/ant-design/node_modules/.pnpm/tsx@4.21.0/node_modules/tsx/dist/esm/index.mjs?1765707025996:2:4984)
    at resolve (file:///Users/lemonade/Downloads/github/ant-design/node_modules/.pnpm/tsx@4.21.0/node_modules/tsx/dist/esm/index.mjs?1765707025996:2:5361)
    at nextResolve (node:internal/modules/esm/hooks:748:28)
    at Hooks.resolve (node:internal/modules/esm/hooks:240:30) {
  code: 'ERR_MODULE_NOT_FOUND',
  url: 'file:///Users/lemonade/Downloads/github/ant-design/scripts/check-commit.ts'
}

Node.js v22.15.0
 ELIFECYCLE  Command failed with exit code 1.
➜  ant-design git:(master) 

| Environment | Info |
| --- | --- |
| antd | 6.1.0 |
| React | 19 |
| System | macos |
| Browser | chrome143.0.7499.41 (正式版本) (arm64)  |

---
如果不需要则删除脚本，且test-all.sh中也有使用check-commit脚本。如果需要删除或者其他更改我可以试试然后提交pr谢谢。

<!-- generated by ant-design-issue-helper. DO NOT REMOVE -->

--- a/package.json
+++ b/package.json
@@ -45,7 +45,6 @@
     "authors": "tsx scripts/generate-authors.ts",
     "build": "npm run compile && cross-env NODE_OPTIONS='--max-old-space-size=4096' npm run dist",
     "changelog": "npm run lint:changelog && tsx scripts/print-changelog.ts",
-    "check-commit": "tsx scripts/check-commit.ts",
     "clean": "antd-tools run clean && rimraf es lib coverage locale dist report.html artifacts.zip oss-artifacts.zip",
     "clean:lockfiles": "rimraf package-lock.json yarn.lock",
     "precompile": "npm run prestart",

--- a/scripts/test-all.sh
+++ b/scripts/test-all.sh
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 # Full skip argms
-# npm run test-all -- --skip-changelog --skip-commit --skip-lint --skip-build --skip-dekko --skip-dist --skip-es --skip-lib --skip-test --skip-node
+# npm run test-all -- --skip-changelog --skip-lint --skip-build --skip-dekko --skip-dist --skip-es --skip-lib --skip-test --skip-node
 
 # Check exist argument
 has_arg() {
@@ -25,14 +25,6 @@ if ! has_arg '--skip-changelog' "$@"; then
   tsx ./scripts/check-version-md.ts
 else
   echo "[TEST ALL] test changelog...skip"
-fi
-
-if ! has_arg '--skip-commit' "$@"; then
-  echo "[TEST ALL] check-commit"
-  echo "[TEST ALL] check-commit" > ~test-all.txt
-  npm run check-commit
-else
-  echo "[TEST ALL] check-commit...skip"
 fi
 
 if ! has_arg '--skip-lint' "$@"; then

